tasklist/client/e2e/docs/docs-snapshots.spec.ts

```diff
@@ -0,0 +1,521 @@
+    await expect(page).toHaveScreenshot();
+  });
+
+  test('claimed a task', async ({page, taskDetailsPage}) => {
+    const alteredTaskList = [
+      ...MOCK_TASKLIST.slice(0, 1),
+      {...MOCK_TASKLIST[1], assignee: 'demo'},
+      ...MOCK_TASKLIST.slice(2),
+    ];
+    await mockTaskSearch(page, alteredTaskList);
+    await mockTask(
+      page,
+      '1',
+      {...alteredTaskList[1], assignee: 'demo'},
+      [
+        {
+          name: 'passengerName',
+          value: '"ARRON A. ARRONSON"',
+          isValueTruncated: false,
+          previewValue: '""',
+          draft: null,
+        },
+      ],
+      registerPassengerForm,
+    );
+
+    await taskDetailsPage.goto('1');
+
+    await expect(page.getByTestId('tasks-skeleton')).not.toBeVisible();
+
+    await expect(page.getByText('Register Passenger')).toBeVisible();
+    await expect(page.getByText('Name: ARRON A. ARRONSON')).toBeVisible();
+
+    await expect(page.getByText('Complete Task')).toBeVisible();
+    await expect(page.getByText('Complete Task')).toBeEnabled();
+
+    await expect(page).toHaveScreenshot();
+  });
+
+  test('completed a task', async ({page, taskPanelPage}) => {
+    const completedTaskList = [
+      {
+        ...MOCK_TASKLIST[1],
+        assignee: 'demo',
+        taskState: 'COMPLETE',
+        completionDate: now,
+      },
+    ];
+    await mockTaskSearch(page, completedTaskList);
+    await mockTask(
+      page,
+      '1',
+      {...completedTaskList[0], assignee: 'demo'},
+      [
+        {
+          name: 'passengerName',
+          value: '"ARRON A. ARRONSON"',
+          isValueTruncated: false,
+          previewValue: '""',
+          draft: null,
+        },
+      ],
+      registerPassengerForm,
+    );
+
+    await taskPanelPage.goto({filter: 'completed', sortBy: 'completion'});
+
+    await expect(page.getByTestId('tasks-skeleton')).not.toBeVisible();
+
+    await taskPanelPage.openTask(completedTaskList[0].name);
+
+    await expect(page.getByText('Register Passenger')).toBeVisible();
+    await expect(page.getByText('Name: ARRON A. ARRONSON')).toBeVisible();
+
+    await expect(page.getByText('Complete Task')).toBeVisible();
+    await expect(page.getByText('Complete Task')).not.toBeEnabled();
+    await page.getByTestId('scrollable-list').click({position: {x: 0, y: 300}}); // Deselect the button
+
+    await expect(page).toHaveScreenshot();
+  });
+
+  test('processes list', async ({page, processesPage}) => {
+    await mockProcessSearch(page, MOCK_PROCESSES);
+
+    await processesPage.goto();
+
+    await expect(page).toHaveScreenshot();
+  });
+
+  test('processes search', async ({page, processesPage}) => {
+    await mockProcessSearch(page, MOCK_PROCESSES);
+
+    await processesPage.goto();
+    await processesPage.searchForProcess('order');
+    await expect(async () => {
+      expect(await page.getByTestId('process-tile').all()).toHaveLength(1);
+    }).toPass();
```

Comment:
Why you don't want to retry this one?

Reply:
`toHaveLength` isn't retryable on its own so you need to wrap it. Debouncing on the search field means it takes a small amount of time to propagate.